src/App.vue

```diff
@@ -513,7 +513,7 @@ export default class App extends Mixins(
   /** Called to init main entity identifier. */
   private assignIdentifier (): void {
     // Capture identifier from query param
-    const id = this.$route.query?.id as string
+    const id: string = this.$route.query?.id
```

Comment:
I think it's easier to see this way what type this variable is (instead of having the type at the end).

This is a really small thing but since I was in cleanup mode...